composer.json

```diff
@@ -19,7 +19,7 @@
         }
     ],
     "require": {
-        "php": ">=7.0",
+        "php": ">=7.1",
```

Comment:
Should probably be `^7.1 || ~8.0.0 || ~8.1.0`

Reply:
Really? Isn't PHP 8.2 supposed to be compatible with PHP 8.1?

Reply:
PHP doesn't (can't) follow SemVer, but in general, the current range selector also allows for PHP 9, which is a problem.

`^7.1 || ^8.0` is also acceptable, if you feel like you aren't worried about breakages in 8.2+

Reply:
I started a PR and would appreciate your input on this: https://github.com/cocur/slugify/pull/301